### ISSUE TITLE
Обсуждение задачи. Вопросы Дмитрию

### ISSUE THREAD

**Issue by svtarasov** [lanit-summer/cleangames#14]
Мне кажется всем будет удобно появляющиеся вопросы сразу писать здесь, а Дмитрию будет удобно их смотреть в одном месте.

**Comment by Mikelle** [comment]
После авторизации указывается ли где-нибудь имя пользователя?

**Comment by svtarasov** [comment]
Дима, скажи пожалуйста, какие виды мусора будут в ближайшей игре и сколько их будет?

**Comment by svtarasov** [comment]
Так виды мусора на текущую игру следующие: 
Пластик
Металл
Стекло
Смешанный мусор
Батарейки

**Comment by mtr1305** [comment]
> После авторизации указывается ли где-нибудь имя пользователя?

Да, это стоит сделать. я об этом не думал, но надо бы. Впрочем, все эти вещи выскакивают тогда, когда начинаешь делать мокапы.
